app/assets/javascripts/pages.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
err, I realise I should made this commit into master, but I removed turobo links and coffee script, so eventually these will have to be changed to `.js` rather than `.coffee`. mostly my bad